src/Core/Utils/SystemUtils.hpp

```
@@ -93,4 +101,55 @@ namespace SystemUtils {
 
         return static_cast<void*>(base + offset);
     }
+
+
+    struct VkLoaderDiag {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: present

(cppcoreguidelines-pro-type-member-init)